src/js/popup/popup.ts

```diff
@@ -294,7 +294,7 @@ let currentOrigin: Origin;
   const params = new URL(document.location.href).searchParams;
   setUpBackgroundMessageHandler(params.get('tab_id'));
   const state = params.get('state') as State;
-  state && updateDisplay(state);
```

Comment:
Why are we removing the state check here?

Reply:
Redundant because of the line above, eslint was complaining.